pkg/image/api/validation/validation.go

```diff
@@ -308,6 +308,9 @@ func ValidateImageStreamImport(isi *api.ImageStreamImport) field.ErrorList {
 			if len(spec.From.Name) == 0 {
 				errs = append(errs, field.Required(imagesPath.Index(i).Child("from", "name"), ""))
 			} else {
+				if spec.From.Name == "*" {
```

Comment:
The problem is that `ParseDockerImageReference` the `*` is no longer valid name. Clients use this character to look up Docker images.